src/main/java/View/InMemoryUser.java

```diff
@@ -0,0 +1,38 @@
+package View;
+
+import ViewModel.PlaylistDsView;
+import ViewModel.UserDsView;
+
+import java.util.ArrayList;
+
+public class InMemoryUser implements UserDsView {
```

Comment:
This solves the issue we had with view unable to access user information since that would violate the clean architecture. Originally we planned to put these data in the loginoutputData, but having this class would work better. Thank you for making this.